package.json

```diff
@@ -41,25 +41,26 @@
     "js-combinatorics": "^0.5.2",
     "lodash": "^4.17.4",
     "mz": "^2.6.0",
-    "request-promise": "^4.2.0",
-    "require-dir": "^0.3.1",
-    "uuid": "^3.0.1",
+    "request-promise": "^4.2.1",
+    "require-dir": "^0.3.2",
+    "uuid": "^3.1.0",
     "winston": "^2.3.1"
   },
   "devDependencies": {
-    "chai": "3.5.0",
-    "chai-as-promised": "6.0.0",
+    "chai": "4.0.2",
```

Comment:
if devDependencies are being added to npm-shrinkwrap.json then should the carets ('^') be added like the are for the normal dependencies?